test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -109,42 +109,20 @@ func TestWorkspacesVirtualWorkspaces(t *testing.T) {
 				vwUser2Client := server.virtualWorkspaceClients[1]
 				workspace1, err := vwUser1Client.TenancyV1beta1().Workspaces().Create(ctx, testData.workspace1.DeepCopy(), metav1.CreateOptions{})
 				require.NoError(t, err, "failed to create workspace1")
-
+				_, err = server.orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, workspace1.Name, metav1.GetOptions{})
```

Comment:
Oh... I was missing that a workspace was created, and the cluster workspace was expected to be automatically created as a result?  That implies ClusterWorkspace is created asynchronously, in which case a poll loop _is_ suggested. Apologies for my confusion.  Maybe add a comment in case any future maintainers also miss this detail?

Reply:
In this PR, Stefan has separated our v1alpha1 Workspace into 2 things: a v1alpha1 ClusterWorkspace, and a v1beta1 Workspace.

The ClusterWorkspace is subject to the possibility of more change (due to its alpha nature), and contains things we don't want or need users to see: initializers, location (shard) information. These CRs are stored in etcd.

The Workspace is intentionally minimal, containing only type, URL, and phase. These are fields we are comfortable with/confident in, at the beta level (ideally won't change). These are not stored in etcd and are available as virtual projections coming from a ClusterWorkspace.

When you use the "personal workspace virtual workspace apiserver" to create a Workspace, the underlying apiserver code syncrhonously creates a ClusterWorkspace + some RBAC pieces, then returns to the client a Workspace, with fields filled in from the real ClusterWorkspace.

Reply:
That's helpful, thank you. I'm not sure this was adequately captured in workspaces.md, maybe a follow-up is suggested?

Reply:
the type is different. The user API is Workspaces, they backed by ClusterWorkspaces. This code checks that the name is the same.

Reply:
Maybe add a comment ensuring this detail is obvious to future maintainers?

Reply:
added